File: Java/Estrutura repetitiva/Enquanto/Media_idades.java
import java.util.Locale;
import java.util.Scanner;

public class Media_idades {
    public static void main(String[] args) {

        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        int idade, cont, soma;
        double media;

        System.out.println("Digite as idades: ");
        idade = sc.nextInt();

        cont = 0;
        soma = 0;
        if (idade < 0) {
            System.out.println("IMPOSSIVEL CALCULAR");
        } else {
            while (idade >= 0) {
                cont = cont + 1;
                soma = soma + idade;
                idade = sc.nextInt();
            }
            media = (double) soma / cont;

            System.out.println("MEDIA = " + String.format("%.2f", media));
        }

        sc.close();

    }
}

File: Python/Estrutura Repetitiva/Enquanto/Media_idades.py
idades: int; soma: int; cont: int
media: float

idades = int(input("Digite as idades: \n"))

if idades < 0:
    print(f"IMPOSSIVEL CALCULAR")
else:
    cont = 0
    soma = 0
    while idades >= 0:
        soma = soma + idades
        cont = cont + 1
        idades = int(input())

    
    media = soma / cont
    print(f"MEDIA = {media:.2f}")




    

    